### ISSUE TITLE
Would be awesome to be able to push a set of helpers to the @preload

### ISSUE THREAD

**Issue by Vincz** [halt-hammerzeit/react-isomorphic-render#18]
It would be great to be able to pass additional args (or an helpers objects) to the preload.
Just like https://github.com/makeomatic/redux-connect

**Comment by halt-hammerzeit** [comment]
I don't get what you mean

**Comment by Vincz** [comment]
Redux connect is just like your preload annotation, it allows to prefetch data before entering a route.
By default, the preload callback has this signature:

    @preload(function(dispatch, getState, location, params) { return Promise })

I suggest to add a configurable helpers argument. The signature would be:

    @preload(function(dispatch, getState, location, params, helpers) { return Promise })

And in the configuration we could add an option like that to configure the helpers:

    preload: {
        helpers: {
                logger: (msg) => console.log("I prefetched :"+msg)
        }
    }

So in my preload method, i could do something like:

    @preload((dispatch, helpers) => {
        dispatch(...some_action...);
        helpers.logger("A little log");
    })

**Comment by halt-hammerzeit** [comment]
Ok, and what is `params` in `redux-connect`?

**Comment by Vincz** [comment]
It is the route params.
If my route is defined with something like that

    <Route path="/page/:slug">

I would get the slug in params

**Comment by halt-hammerzeit** [comment]
Implemented in `5.0.0`.
```js
{
  preload:
  {
    helpers:
    {
      logger: ...
    }
  }
}
```
